nservicebus/hosting/ibus-instance.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Managing IBus instance
+summary: Fluent configuration API to get transnational one-way messaging
```

Comment:
sp: transactive (not transnational)
summary: Configuring the application for acquiring the IBus instance